addons/binding/org.openhab.binding.gardena/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,521 @@
+        <description>The internal temperature of the robotic mower</description>
+        <channels>
+            <channel id="temperature" typeId="temperature" />
+        </channels>
+    </channel-group-type>
+
+
+    <channel-type id="mowerCommandDuration">
+        <item-type>Number</item-type>
+        <label>Mowing time</label>
+        <description>Mowing time in minutes</description>
+        <state pattern="%d min" min="1" max="1440" step="1" />
+    </channel-type>
+
+    <channel-type id="parkUntilFurtherNotice">
+        <item-type>Switch</item-type>
+        <label>Park And Pause All Timers</label>
+        <description>Park and pause all timers</description>
+    </channel-type>
+
+    <channel-type id="parkUntilNextTimerCommand">
+        <item-type>Switch</item-type>
+        <label>Park Until Next Timer</label>
+        <description>Park until next timer</description>
+    </channel-type>
+
+    <channel-type id="startResumeSchedule">
+        <item-type>Switch</item-type>
+        <label>Start Resume Schedule</label>
+        <description>Start resume schedule</description>
+    </channel-type>
+
+    <channel-type id="startOverrideTimer">
+        <item-type>Switch</item-type>
+        <label>Override Timer</label>
+        <description>Starts the mower for the specified duration</description>
+    </channel-type>
+
+    <channel-type id="lastTimeOnline" advanced="true">
+        <item-type>DateTime</item-type>
+        <label>Last Time Online</label>
+        <description>Last time the device was online</description>
+        <state readOnly="true" />
+    </channel-type>
+
+    <channel-type id="rechargeableBatteryStatus" advanced="true">
+        <item-type>String</item-type>
+        <label>Rechargeable Battery Status</label>
+        <description>The status of the rechargeable battery</description>
+        <state readOnly="true">
+            <options>
+                <option value="weak">Weak</option>
+                <option value="ok">OK</option>
+                <option value="undefined">Undefined</option>
+            </options>
+        </state>
+    </channel-type>
+
+    <channel-type id="charging" advanced="true">
+        <item-type>Switch</item-type>
+        <label>Battery Charging</label>
+        <description>Battery Charging</description>
+        <state readOnly="true" />
+    </channel-type>
+
+    <channel-type id="radioQuality" advanced="true">
+        <item-type>Number</item-type>
+        <label>Radio Link Quality</label>
+        <description>Radio Link Quality</description>
+        <state pattern="%d %%" readOnly="true" />
+    </channel-type>
+
+    <channel-type id="connectionStatus">
+        <item-type>String</item-type>
+        <label>Connection Status</label>
+        <description>Connection Status</description>
+        <state readOnly="true">
+            <options>
+                <option value="status_device_unreachable">Device unreachable</option>
```

Comment:
again ... sitemap 😉  I hope the new sitemaps will be ready soon, for me it's the most missing feature, that i can not show the Thing status/properties